scalableminds/webknossos
# Fix bug where upload-hint wasn't shown in dashboard

… under certain circumstances

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set `isDemoInstance` to true in `application.conf`
- Make one dataset public
- Log out and create a fresh account via the landing page
- Go to "My Datasets"
- The screen should look like this:
![image](https://user-images.githubusercontent.com/2486553/113993109-5ffca600-9854-11eb-9a42-17c381ef6e07.png)
- I also checked the network tab to ensure that the `datasets` route delivers the public dataset. however, the dataset won't be shown, as the "my datasets" tab only shows datasets of one's own organization.


### Issues:
- fixes #5364

------
